src/pages/p2p/components/_p2pbanner.tsx

```diff
@@ -173,11 +165,26 @@ const P2PBanner = ({ title }: P2PBannerProps) => {
                 />
             </StyledText>
             <Desktop>
-                <Background
-                    style={{ height: '340px', backgroundPosition: '20% 20%' }}
-                    data={is_rtl ? data['p2p_banner_rtl'] : data['p2p_banner']}
-                >
-                    <Wrapper>
+                <BackgroundImageWrapper>
+                    <StaticImageWrapper>
+                        {is_rtl ? (
```

Comment:
same here

Reply:
We can't pass dynamic images or variable to `StaticImage`